pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
something like the getty stuff should probably be in the top-level docs. It's fine to have packages specific docs, but I wouldn't expect them to change much (unless the code changes) so shouldn't be a problem re hash